cachelib/allocator/MM2Q.h

```diff
@@ -213,7 +216,8 @@ class MM2Q {
            bool rebalanceOnRecordAccs,
            size_t hotPercent,
            size_t coldPercent,
-           uint32_t mmReconfigureInterval)
+           uint32_t mmReconfigureInterval,
+           bool useCombinedLockForIterators)
```

Comment:
Please make this optional. 
The way we typically do is create a new constructor with all the parameters while keeping the constructor of ```Config(uint32_t time,
           double ratio,
           bool updateOnW,
           bool updateOnR,
           bool tryLockU,
           bool rebalanceOnRecordAccs,
           size_t hotPercent,
           size_t coldPercent,
           uint32_t mmReconfigureInterval)```
And have it call the constructor with all the parameters with default value.

Reply:
Sure, done.